src/main/java/jarvis/utils/Storage.java

```diff
@@ -0,0 +1,95 @@
+package jarvis.utils;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+
+import jarvis.exceptions.InvalidTaskException;
+import jarvis.tasks.Task;
+
+public class Storage {
+    private static final String DELIMITER = ",";
+
+    private static final String ROOT_FOLDER = System.getProperty("user.dir");
+    private static final String DATA_DIR = "data";
+    private static final String FILENAME = "duke.txt";
+    private static final Path FILE_PATH = Paths.get(ROOT_FOLDER, DATA_DIR, FILENAME);
+
+    public Storage() {
```

Comment:
Default constructors are automatically generated, would it be better to remove this line ?